functions/signups.js

```diff
@@ -109,9 +109,10 @@ z           }
         }
 
         let logMessage = 'Sign Up: ' + characterName + ' => ' + signValue;
+        client.log.raidLog(client, message, characterName + " signed up as " + signValue + ".", "Signed up as " + signValue);
         client.log.write(client, message.author, message.channel, logMessage);
```

Comment:
Since there is more than 1 type of log being generated here in 2 lines, the client.log.write could be updated in name to reflect the type of logging being done.